frontend/src/APIClients/queries/CourseQueries.ts

```diff
@@ -0,0 +1,29 @@
+import { gql } from "@apollo/client";
+
+export const COURSES = gql`
+  query Courses {
+    courses {
+      title
+      description
+      image
+      previewImage
+      lessons
```

Comment:
Courses now contain modules, not lessons (same for the GET_COURSE graphql string)